packages/app/src/search/Search.module.css

```diff
@@ -5,14 +5,24 @@
 
 .inputRow {
   display: flex;
-  flex-direction: column;
+  align-items: center;
+  height: var(--toolbar-height);
+  padding: 0 0.5rem;
 }
 
-.inputRow input {
+.input {
+  flex: 1 1 0%;
+  height: var(--toolbarBtn-height);
+  padding: 0 0.5rem;
+  border: none;
   border-radius: 0.5rem;
-  margin: 0.625rem;
-  padding: 0.25rem;
-  box-shadow: none;
+  box-shadow: 0 0 2px dimgray;
+  outline: none;
```

Comment:
Replacing the outline with a box-shadow for cross-browser consistency and for consistency with toolbar inputs (i.e. cell width in Matrix vis and bound editors in domain tooltip). That being said, I use a "flat" shadow instead of a "raised" shadow, since the sidebar's styles are "flatter" that the toolbar's.